Revise notes about Raft Engine (#7995)
pingcap/docs: tikv-configuration-file.md

Configuration items related to Raft Engine.

> **Note:**
>

> > - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds to start TiKV.

Reviewer: ```suggestion
> - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds for TiKV to start.
```

> > - When you enable Raft Engine for the first time, TiKV transfers its original data from RocksDB to Raft Engine. Therefore, you need to wait an extra tens of seconds to start TiKV.

Author: ```suggestion
> - When you enable Raft Engine for the first time, TiKV transfers its data from RocksDB to Raft Engine. Therefore, you need to wait extra tens of seconds to start TiKV.
```

> > - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, if you need to downgrade a TiDB cluster from v5.4.0 to earlier versions, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restarting TiKV for the configuration to take effect.

Reviewer 2: ```suggestion
> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with earlier TiDB versions. Therefore, if you need to downgrade a TiDB cluster from v5.4.0 to an earlier version, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restart TiKV for the configuration to take effect.
```

> > - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, when you need to downgrade a TiDB cluster from v5.4.0 to earlier versions, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restarting TiKV for the configuration to take effect.

Author: ```suggestion
> - The data format of Raft Engine in TiDB v5.4.0 is not compatible with previous versions. Therefore, if you need to downgrade a TiDB cluster from v5.4.0 to earlier versions, **before** downgrading, disable Raft Engine by setting `enable` to `false` and restarting TiKV for the configuration to take effect.
```
